demo/components/victory-tooltip-demo.js

```diff
@@ -172,4 +175,94 @@ class App extends React.Component {
   }
 }
 
+class VictoryTooltipInPortal extends VictoryTooltip {
+  static contextTypes = {
+    portalRegister: React.PropTypes.func,
+    portalDeregister: React.PropTypes.func
+  }
+
+  render() {
+    this.element = super.render();
+    return null;
+  }
+
+  componentDidUpdate() {
+    this.context.portalRegister(this, this.element);
+  }
+
+  componentWillUnmount() {
+    this.context.portalDeregister(this);
+  }
+}
+
+class SvgPortal extends React.Component {
+  componentWillMount() {
+    this.map = new Map();
+    this.portalRegister = this.portalRegister.bind(this);
+    this.portalDeregister = this.portalDeregister.bind(this);
+  }
+
+  portalRegister(key, element) {
+    this.map.set(key, element);
+    this.forceUpdate();
+  }
+
+  portalDeregister(key) {
+    this.map.delete(key);
+  }
+
+  render() {
+    return (
+      <g>
+        {[...this.map.values()].map((el, i) => {
+          return el ? React.cloneElement(el, {key: i}) : el;
+        })}
+      </g>
+    );
+  }
+}
+
+class VictoryContainer extends React.Component {
```

Comment:
Actually I prefer either 1) creating a new component that extends VictoryContainer or 2) adding all this functionality directly to VictoryContainer and turning it on / off with props. I'm imagining something like 
```
<VictoryChart containerComponent={<VictoryContainer portalName={"myCoolPortal"}/>}>
  ... portal name gets passed to children and labels end up on the portal if it exists
</VictoryChart>
```

Feasible?